egress/README.md

```diff
@@ -99,59 +99,47 @@ application using SAM!
 ## Build
 
 ```
-sam build -m requirements.txt --use-container
+sam build --use-container
 ```
 
-### Local development
-
-SAM provides a lot of capability to test and run your system locally before
-deploying AWS resources.
-
-```
-sam local invoke -e event.json --env-vars env.json
-```
-
- - `sam local invoke` invokes the Lambda function using docker locally
- - `event.json` is a mock request object to send to the Lambda function
- - `env.json` allows a user to specify a local set of environment variables.
- See the provided `env.json-example` for an example.
+## Deploy
 
-You can also run the Lambda on a local API server and send full web-requests
-by running:
+Before we can deploy anything, we need an S3 bucket where we can upload our
+Lambda function packaged as a ZIP file. If you don't have an S3 bucket to store
+code artifacts, then this is a good time to create one:
 
-```
-sam local start-api --port 5000 --env-vars env.json --host 0.0.0.0
+```bash
+BUCKET=BUCKET_NAME
+aws s3 mb s3://${BUCKET}
```

Comment:
You need to make sure they have the aws cli, or is that done elsewhere?

Reply:
Further up in this readme, not in the diff